### PR TITLE
Stackrox release 3.72

### PR DESCRIPTION
**Pull request by parametalol**
CI configuration files to support Stackrox release 3.72.

### CODE REVIEW

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.72.yaml [hunk 1]
```diff
@@ -0,0 +1,367 @@
+base_images:
+  eks:
+    name: automation-flavors
+    namespace: stackrox
+    tag: eks-0.3.1
+  golang-1.17.2:
+    name: golang
+    namespace: stackrox
+    tag: 1.17.2
```
Comment (vikin91) [inline ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.72.yaml:9]: Isn't this a bit too old? We run on 1.17.12 or 13 now, right?

Comment (0x656b694d) [reply ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.72.yaml:9]: That's the version used in nightlies. Do we want to bump it now for the release?

Comment (vikin91) [reply ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.72.yaml:9]: Just spotted this, so not sure how much it affects the release. Nightlies on the release branch sound strange. Isn't it the builder that would build the upstream released version?

Comment (vikin91) [reply ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.72.yaml:9]: Let's maybe ask @gavin-stackrox what this version mens to learn for the future.

The PR is good to be merged for me 👍

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.72.yaml [hunk 2]
```diff
@@ -0,0 +1,367 @@
+base_images:
+  eks:
+    name: automation-flavors
+    namespace: stackrox
+    tag: eks-0.3.1
+  golang-1.17.2:
```
Comment (gavin-stackrox) [inline ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.72.yaml:6]: This needs to be `1.17.12` to match operator/Dockerfile on the target branch. (and in the 3 other places in this file). 
Marcin recently updated it, so you may have used a slightly stale openshift/release...master.yaml as the template.

**Review by gavin-stackrox — CHANGES_REQUESTED** [review]

**Pull request merged**